### PR TITLE
Feature/nh user flow tweaks

### LINKED ISSUE #64: Hide things when logged out
switch sign-in button to logout

### LINKED ISSUE #66: Add log out function

### PR DESCRIPTION
**Pull request by nickhartdev**
#### What's this PR do?
- increases writing time to 2 minutes
- adds Sign Out conditional render
#### Where should the reviewer start?
- Header.js
#### How should this be manually tested?
Log in, start a story. You should see a log out button on the way, and the timer should start at 2 minutes.
#### What are the relevant tickets?
Resolves #66, resolves #64

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -109,6 +109,10 @@ class App extends Component {
     this.checkStoryData(this.state.stories)
   }
 
+  signOut = () => {
```
Comment (GreysonElkins) [inline src/components/App/App.js:112]: beautiful

File: src/components/TimerDisplay/TimerDisplay.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import Timer from 'react-compound-timer'
 const TimerDisplay = (props) => {
   return(
     <Timer 
-      initialTime={59000} 
+      initialTime={119000}
```
Comment (GreysonElkins) [inline src/components/TimerDisplay/TimerDisplay.js:7]: THAT'S A LOT OF TIME

File: src/components/TimerDisplay/TimerDisplay.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const TimerDisplay = (props) => {
       ]}
     >
       <Fragment>
-        <Timer.Seconds />
+        <Timer.Minutes />:<Timer.Seconds />
```
Comment (GreysonElkins) [inline src/components/TimerDisplay/TimerDisplay.js:17]: Cooooool

**Review by GreysonElkins — APPROVED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (Abdeboskey) [reply src/components/App/App.js:112]: So simple, so elegant.

File: src/components/TimerDisplay/TimerDisplay.js (hunk 2 above)
Comment (Abdeboskey) [reply src/components/TimerDisplay/TimerDisplay.js:7]: That's just 2 big ones.

File: src/components/TimerDisplay/TimerDisplay.js (hunk 3 above)
Comment (Abdeboskey) [reply src/components/TimerDisplay/TimerDisplay.js:17]: Such a sick plugin.

**Review by Abdeboskey — APPROVED** [review]
All about it, Thank you!

File: src/components/App/App.js (hunk 1 above)
Comment (carlymclift) [reply src/components/App/App.js:112]: So great!

File: src/components/TimerDisplay/TimerDisplay.js (hunk 2 above)
Comment (carlymclift) [reply src/components/TimerDisplay/TimerDisplay.js:7]: 👍🏼

**Review by carlymclift — APPROVED** [review]
I approve!

**Pull request merged**